### ISSUE TITLE
Feature request: Playback speed

### ISSUE THREAD

**Issue by Primokorn** [timusus/Shuttle#41]
Hi,
Thanks for opening the source code for your app!
Could you please add a playback speed feature to Shuttle? I'm used to listening to my podcasts at x1.50.
I know [VLC](https://code.videolan.org/videolan/vlc-android) has this feature.

Thanks anyways.

**Comment by timusus** [comment]
Definitely worth considering.

Thinking out loud, I would want this to be something that's optional and off by default. 

Figuring out how to present this nicely is a challenge too.

**Comment by Primokorn** [comment]
I use it daily. IMHO it should be added on the screen of the current played track (better than in the general settings).
![screenshot_20170228-095201](https://cloud.githubusercontent.com/assets/5938828/23407142/c26cf960-fdc2-11e6-83a4-98bced0f60b4.png)

FYI [AntennaPod](https://github.com/antennapod/AntennaPod) also has this feature through its built-in player.

**Comment by timusus** [comment]
Right, but we need to consider that the average Shuttle user probably never uses this feature at all. 

The ui control to adjust the speed should definitely be on the now playing screen, but only if it's enabled via settings.

**Comment by Primokorn** [comment]
Sure, that's perfect.
